jobs.groovy

```diff
@@ -122,18 +122,17 @@ GLOBAL_CONFIG = jsonSlurper.parseText(
 // --------------------------------------------------------------------------
 
 def updateGitHubStatus(status, description, branch, dashProject,
-                       dashBranchName) {
+                       dashBranchName, jobName) {
     return """#!/bin/bash
 # Jenkins jobs are usually run using 'set -x' however we use 'set +x' here
 # to avoid leaking credentials into the build logs.
 set +x
-# Read the GitHub credentials
-. /etc/slave_config/github_status.sh
-
+# Set the GitHub credentials
+""" + new File('/etc/slave_config/github_status.sh').text + """
```

Comment:
We embed this script because it's not available on the slaves, where the success/failure notification will now be sent from.